src/cljs/rems/administration/create_catalogue_item.cljs

```diff
@@ -188,13 +188,34 @@
          :item-selected? item-selected?
          :on-change #(rf/dispatch [::set-selected-workflow %])}])]))
 
+(defn resourse-dropdown [r language counts]
+  (let [organisation? (get-in r [:organization :organization/short-name (keyword language)])
```

Comment:
I'm pretty sure the language is guaranteed to be a keyword, I think you can just replace `(keyword language)` with `language`.